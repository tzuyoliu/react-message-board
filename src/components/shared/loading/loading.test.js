import React from 'react'
import { create } from 'react-test-renderer'
import Loading from '~/components/shared/loading'

describe('loading.test.js', () => {
    test('<Loading /> should render', () => {
        const loading = create(<Loading />)
        expect(loading).toBeTruthy()
    })

    test('<Loading /> should match the snapshot', () => {
        const loading = create(<Loading>Snapshot Test</Loading>)
        expect(loading.toJSON()).toMatchSnapshot()
    })
})
