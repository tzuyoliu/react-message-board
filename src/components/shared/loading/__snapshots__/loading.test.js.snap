// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading.test.js <Loading /> should match the snapshot 1`] = `
<div
  className="sc-bdVaJa fJXfGw"
>
  Snapshot Test
</div>
`;
